src/react-components/room/useAvatar.js

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from "react";
+
+export default function useAvatar() {
+  const [state, setState] = useState({ hasWebcamTextureTarget: false });
+
+  useEffect(() => {
+    const avatarModelEl = document.querySelector("#avatar-rig .model");
```

Comment:
Does this `querySelector` have to run again when someone changes their avatar?

Reply:
No it doesn't. The `model` class is on the entity with the `gltf-model-plus` component which is never removed. So this query will only be ran when the React component mounts.